src/snapred/backend/data/LocalDataService.py

```diff
@@ -1081,7 +632,22 @@ def initializeState(self, runId: str, useLiteMode: bool, name: str = None):
                 seedRun=runId,
                 useLiteMode=liteMode,
                 creationDate=datetime.datetime.now(),
-                version=self.VERSION_START,
+                version=version,
+            )
+            record = Record(
+                runNumber="default",
+                useLiteMode=liteMode,
+                version=version,
+                calculationParameters=calibration,
+            )
+            entry = IndexEntry(
+                runNumber="default",
+                useLiteMode=liteMode,
+                version=version,
+                appliesTo=">=0",
+                author="SNAPRed Internal",
+                comments="The default condition for loading StateConfigs if none other found",
+                timestamp=round(time.time()),
```

Comment:
The `Record` and `IndexEntry` are necessary for the `Indexor` to find the "CalibrationParameters.json" file.